docs/services/compute/virtual-machines.md

```diff
@@ -42,23 +42,140 @@ An Azure virtual machine gives you the flexibility of virtualization without hav
+
+Once tag inheritance is enabled, tags from billing profiles, invoice sections, subscriptions, and resource groups are automatically applied to child resource usage records for the current month within 24 hours. For detailed instructions, refer to the [Cost allocation guidance](https://learn.microsoft.com/en-us/azure/cost-management-billing/costs/enable-tag-inheritance).
+
+#### Tagging Best Practices:
+
+- **Consistent Tagging:** Establish a consistent tagging strategy across VMs using the [Cloud Adoption Framework (CAF) guidance](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-tagging).
+
+- **Automated Tagging with Azure Policy:** Simplify and standardize tagging by utilizing Azure Policy to automatically apply tags to VMs. This streamlines the allocation process, ensuring adherence to tagging standards.
+
+Implementing tag inheritance and following these tagging best practices enhances your ability to efficiently allocate and manage VM costs.
+
 ### Reporting + analytics
 
+Effectively managing and optimizing costs associated with your Virtual Machines (VMs) in Azure requires a strategic approach to reporting and analytics. Services such as Cost Management and Azure Monitor play a crucial role in analyzing VM usage and expenditures.
+
+To report on VM costs comprehensively, consider all the services mentioned in the Data Ingestion capability, including but not limited to:
+
+- **Virtual Machines:** Understand the hourly billing, VM size, regional pricing, and states (running, stopped, deallocated).
+- **Storage:** Factor in storage costs based on disk size and transactions per hour.
+- **Virtual Network and Bandwidth:** Include costs related to data transfer and bandwidth usage.
+- **Associated Services:** Consider costs associated with Azure Monitor, Azure Bastion, Azure DNS, and Load Balancer if used.
+
+## Validating VM Usage:
+
+Once you've identified all the costs associated with a VM, the next crucial step is to validate the VM usage to ensure it is properly sized. This involves:
+
+- **Right-sizing VMs:** Review the VM size to ensure it aligns with the actual workload requirements. Consider scaling up or down based on usage patterns.
+- **Efficient Resource Management:** Leverage Azure services such as Cost Management and Azure Monitor to gain insights into VM performance metrics, allowing you to identify areas for optimization.
+
+
 ### Anomalies
+Anomaly detection helps pinpoint unexpected spikes or irregularities in VM spending, enabling timely intervention to optimize costs. IIdentifying irregularities helps in optimizing resource allocation, ensuring resources are used efficiently.
```

Comment:
This is all generic. We can probably remove it like I mentioned for allocation above. Maybe every section would have a link to the general FinOps capability guidance 🤔

Reply:
Added placeholders to include the links to the FinOps capability guidance. Will do that in a later PR.